src/execution/compiler/expression/star_translator.cpp

```diff
@@ -0,0 +1,15 @@
+#include "execution/compiler/expression/star_translator.h"
+#include "execution/compiler/translator_factory.h"
+#include "execution/sql/value.h"
+#include "parser/expression/star_expression.h"
+#include "type/transient_value_peeker.h"
+
+namespace terrier::execution::compiler {
+StarTranslator::StarTranslator(const terrier::parser::AbstractExpression *expression, CodeGen *codegen)
+    : ExpressionTranslator(expression, codegen) {}
+
+ast::Expr *StarTranslator::DeriveExpr(ExpressionEvaluator *evaluator) {
+  // TODO(Amadou): What we return here doesn't really matter. Should StarExpression be ConstantValueExpression?
```

Comment:
Return `1`, which is logically what a `COUNT(*)` returns on each tuple. Change comment to "Result technically isn't used. Can be replaced with a constant value expression.` Or something to that effect.